spark.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM imranq2/spark_python:0.1.25
+# https://github.com/imranq2/docker.spark_python
+USER root
+
+ENV PYTHONPATH=/sam
+ENV CLASSPATH=/sam/jars:$CLASSPATH
+
+COPY Pipfile* /sam/
+WORKDIR /sam
```

Comment:
I just noticed the dir name here i.e.`sam` is different from the one defined in the docker-compose file (SparkAutoMapper)